contracts/Pool.sol

```diff
@@ -61,7 +61,7 @@ error UnauthorizedBorrower();
 error BorrowFromEmptyPool();
 
 /// The pool utilisation cannot be greater than 95%
-error PoolUtilisationTooHigh();
+error PoolUtilisationTooHighForBorrow();
```

Comment:
I'd say PoolUtilisationTooHighToBorrow or PoolUtilisationTooHighForBorrowing. What do you think?

Reply:
For Borrowing